components/properties/src/runtime.rs

```diff
@@ -0,0 +1,405 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! 🚧 \[Experimental\] This module is experimental and currently crate-private. Let us know if you
```

Comment:
I also thought that the module name was a little iffy before reading your [previous comment](https://github.com/unicode-org/icu4x/pull/3169#discussion_r1125416585) wondering the same out loud.

Some more options:
- regex
- alias
- parse / alias_parse

"runtime" is somewhat generic as it is used here, which is why I think of the more specific sense (noun) rather than adjective/adverb sense, and the noun sense becomes misleading.

I don't know if UnicodeProperty as an enum would be used anywhere else but here, so I think its presence here is fine even after a module renam.e

Reply:
The module is private for now so the name doesn't matter for now.